models/user.go

```diff
@@ -18,7 +18,8 @@ type User struct {
 	UpdatedAt time.Time  `json:"updated_at"`
 	DeletedAt *time.Time `json:"-"`
 
-	OrderCount int64 `json:"order_count,ommitempty" gorm:"-"`
+	OrderCount  int64      `json:"order_count,ommitempty" gorm:"-"`
+	LastOrderAt *time.Time `json:"last_order_at" gorm:"-"`
```

Comment:
probably want an `omitempty` (https://play.golang.org/p/4LZ4atB1QlK)

Reply:
Is there a convention at Netlify for this?

I usually think `null` is better than a missing field since if the frontend consumes this they will need to check for `null` explicitly. In Javascript this is not important for a falsy check, but in Typescript this is a difference.
@RoseannaM do you prefer `"key" in object` or `object.key !== null`?

Also: There is virtually no case where a user has no order since users in gocommerce are only created once they make an order.